x/bank/v2/module.go

```diff
@@ -102,6 +104,12 @@ func (am AppModule) RegisterMsgHandlers(router appmodulev2.MsgRouter) {
 		errs = errors.Join(errs, err)
 	}
 
+	if err := appmodulev2.RegisterHandler(
```

Comment:
@testinginprod, imho we should improve the API to register multiple messages and handlers in that method. Otherwise it is imho way too verbose.

like 

`appmodulev2.RegisterHandlers(router,
"cosmos.bank.v2.MsgSend": handlers.SendHandler,
"cosmos.bank.v2.MsgUpdateParams": handlers.ParamsHandler)`

It will need to be a bit less type safe I think, but still a better UX.